libraries/apollo-websocket-network-transport-incubating/src/commonMain/kotlin/com/apollographql/apollo3/network/ws/incubating/WebSocketEngine.kt

```diff
@@ -0,0 +1,87 @@
+package com.apollographql.apollo3.network.ws.incubating
+
+import com.apollographql.apollo3.api.http.HttpHeader
+import okio.Closeable
+
+interface WebSocketEngine: Closeable {
+  /**
+   * Creates a new [WebSocket].
+   *
+   * The [WebSocket] is be garbage collected when not used anymore. Call [WebSocket.close] to close the websocket and release resources earlier.
```

Comment:
I'm on a crusade to remove future tense from our docs. It's what our style guide says ([internal link](https://docs.google.com/document/d/1jZhBYPgoZsxe5UEMD3oaGHISoxo6DFR7oJxNij0i6X8/edit)) and I tend to agree despite being guilty of a lot of the current usage.

I'll remove the extra 'be' but I'm in favor of keeping the sentence. IO streams like `InputStream` usually need closing because they hold file descriptors or other resources. Here the file descriptor is released automatically on websocket error or server close. It might become the norm at some point but I don't think we're there yet.